src/main/java/edu/greenblitz/robotName/subsystems/shooter/Pivot/FalconPivot/FalconPivot.java

```diff
@@ -46,42 +67,24 @@ public void resetAngle(Rotation2d position) {
     @Override
     public void moveToAngle(Rotation2d targetAngle) {
         motor.setControl(
-                new MotionMagicDutyCycle(
-                    targetAngle.getRotations() / PivotConstants.RELATIVE_POSITION_CONVERSION_FACTOR,
-                    true,
-                    PIVOT_FEED_FORWARD.calculate(targetAngle.getRadians(), 0),
-                    MOTION_MAGIC_PID_SLOT,
-                    true,
-                    true,
-                    true
-        ));
-    }
-
-    @Override
-    public void standInPlace(Rotation2d targetAngle) {
-        motor.setControl(
-                new PositionVoltage(
-                        targetAngle.getRotations(),
-                        0.0,
-                        true,
-                        PIVOT_FEED_FORWARD.calculate(targetAngle.getRadians(), 0),
-                        STAND_IN_PLACE_PID_SLOT,
-                        true,
-                        true,
-                        true
-                )
+                new PositionVoltage(targetAngle.getRotations())
```

Comment:
create a position voltage at the top of the class like the MK4iSwerveModule class